SBAT.md

```diff
@@ -194,10 +196,11 @@ generation number be incremented past the latest one shown in any
 binary for that product, effectively disabling that product on UEFI
 Secure Boot enabled systems.
 
-A subset of this case are beta-release that may contain eventually
+A subset of this case would be a beta-release that may contain eventually
 abandoned, experimental, kernel code. Such releases should have their
 product-specific generation numbers incremented past the latest one
-shown in any, released or unreleased, production key signed binary.
+shown in any, released or unreleased, binary signed with a production
```

Comment:
Maybe that first comma belongs /after/ "released"?

Reply:
yup